<ui version="4.0" >
 <class>VideoSettings</class>
 <widget class="QWidget" name="VideoSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>505</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="enableVideoCheckBox" >
     <property name="text" >
      <string>Enable video</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="webcamSelectionGroupBox" >
     <property name="title" >
      <string>Webcam Selection</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>356</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Webcam device:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="webcamDeviceComboBox" />
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="webcamPreviewButton" >
        <property name="text" >
         <string>Webcam Preview</string>
        </property>
       </widget>
      </item>
      <item rowspan="4" row="0" column="1" >
       <widget class="QLabel" name="webcamPreviewLabel" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>176</width>
          <height>144</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>176</width>
          <height>144</height>
         </size>
        </property>
        <property name="pixmap" >
         <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/iconbar/webcam.png</pixmap>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Connection Type</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Video quality depends on your Internet connection bandwidth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="videoQualityTreeWidget" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Video quality</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Down (kbit/s)</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Up (kbit/s)</string>
         </property>
        </column>
        <item>
         <property name="text" >
          <string>Normal</string>
         </property>
         <property name="text" >
          <string>0 to 512</string>
         </property>
         <property name="text" >
          <string>0 to 128</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Good</string>
         </property>
         <property name="text" >
          <string>512 to 2048</string>
         </property>
         <property name="text" >
          <string>128 to 256</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Very good</string>
         </property>
         <property name="text" >
          <string>2048+</string>
         </property>
         <property name="text" >
          <string>256+</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Excellent</string>
         </property>
         <property name="text" >
          <string>8192+</string>
         </property>
         <property name="text" >
          <string>1024+</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Note:&lt;/span> When used for the first time we recommend that you select the normal video quality mode, then you can try different modes to find out the most suitable one.&lt;br />&lt;br />&lt;span style=" font-weight:600;">Tip&lt;/span>: Go to http://test.lvcm.com to determine your Internet connection bandwidth.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="makeTestVideoCallButton" >
       <property name="text" >
        <string>Make a test video call</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>562</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>webcamDeviceComboBox</tabstop>
  <tabstop>webcamPreviewButton</tabstop>
  <tabstop>videoQualityTreeWidget</tabstop>
  <tabstop>makeTestVideoCallButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>enableVideoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>webcamSelectionGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>72</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>130</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableVideoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>38</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel" >
     <x>78</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableVideoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>makeTestVideoCallButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
